asabiris/output/pushnotification/service.py

```
@@ -0,0 +1,107 @@
+							tech_message="Push failed: {} {}".format(resp.status, text),
+							error_i18n_key="Push notification failed.",
+							error_dict={"error_message": text}
+						)
+		except aiohttp.ClientError as err:
+			L.error("Network error while sending push: {}".format(err))
+			raise ASABIrisError(
+				ErrorCode.SERVER_ERROR,
+				tech_message="Network error while sending push.",
+				error_i18n_key="Error occurred while sending push. Reason: '{{error_message}}'.",
+				error_dict={"error_message": str(err)}
+			)
```

Ruff (0.14.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)